Zotero/Scenes/Master/MasterContainerViewController.swift

```diff
@@ -16,37 +16,48 @@ protocol DraggableViewController: UIViewController {
     func disablePanning()
 }
 
+protocol BottomSheetObserver: UIViewController {
+    func bottomSheetUpdated(hidden: Bool, containerHeight: CGFloat, topOffset: CGFloat)
+}
+
+extension BottomSheetObserver {
+    func bottomSheetUpdated(hidden: Bool, containerHeight: CGFloat, topOffset: CGFloat) {
+        let bottomInset: CGFloat = hidden ? .zero : (containerHeight - topOffset)
+        additionalSafeAreaInsets = UIEdgeInsets(top: 0, left: 0, bottom: bottomInset, right: 0)
+    }
+}
+
 final class MasterContainerViewController: UINavigationController {
     enum BottomPosition {
         case mostlyVisible
         case `default`
         case hidden
         case custom(CGFloat)
-
+        
```

Comment:
Lots of tabs added again

Reply:
Check https://github.com/zotero/zotero-ios/pull/791/commits/240b793795fe6c73a9740e9a38c0a2278bc72044, still some spaces were removed now, instead of added (a lot less though).